### PR TITLE
feat(deps): Add Nextcloud 28 support on master

### PR DESCRIPTION
**Pull request by nickvergessen**
Signed-off-by: Joas Schilling <coding@schilljs.com>

### CODE REVIEW

**Review by ChristophWurst — COMMENTED** [review]
Commenting only because I'll leave this to the main maintainer

**Review by Pytal — APPROVED** [review]

**Pull request merged**
